todo-app/src/components/CreateTodoList.js

```diff
@@ -0,0 +1,3 @@
+const CreateTodoList = () => {};
```

Comment:
the name of the component should not contain the verb.. like `create`.
It should just be something like `TodoList`